definitions/features/foreman_proxy.rb

```diff
@@ -79,17 +79,28 @@ def config_files_to_exclude(_for_features = ['all'])
 
   def content_module
     return @content_module if @content_module_detected
+
     @content_module_detected = true
     answer = feature(:installer).answers.find do |_, config|
-      config.is_a?(Hash) && config.key?('certs_tar')
+      config.is_a?(Hash) && config.key?(certs_param_name[:param_key])
     end
-    @content_module = answer.nil? ? 'foreman_proxy_content' : answer.first
+    @content_module = answer.nil? ? certs_param_name[:param_section] : answer.first
     logger.debug("foreman proxy content module detected: #{@content_module}")
     @content_module
   end
 
+  def certs_param_name
+    if check_min_version('foreman', '1.21')
+      return { :param_section => 'certs', :param_key => 'tar_file' }
+    end
+
+    { :param_section => 'foreman_proxy_content', :param_key => 'certs_tar' }
+  end
+
   def certs_tar
-    feature(:installer).answers[content_module]['certs_tar'] if content_module
+    if content_module && with_content?
```

Comment:
@upadhyeammit my only concern is if `with_content?` shouldn't be part of the `content_module` method. Does it make sense to have it outside?

Reply:
@upadhyeammit, any updates on this?

Reply:
@mbacovsky I dont see with_content? called in content_module method. Maybe I am unable to understand, can you please elaborate ?

Reply:
@upadhyeammit My thinking was if `content_module` shouldn't imply `with_content?`. If I put it in question - Does it make sense to search for a content_module on a proxy without content?